src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -55,33 +77,54 @@
 helps['account show'] = """
 type: command
 short-summary: Get the details of a subscription.
-long-summary: If no subscription is specified, shows the current subscription.
+long-summary: >
+    If no subscription is specified, information for the current subscription is returned.
+examples:
+    - name: >
+        Get the current default subscription returning results as JSON.  Use `--output yaml` for plain text. 
+        You can also get details of the current subscription by using `az account list --query "[?isDefault]"`.
```

Comment:
It works, yes, but it is too complicated for a simple task like showing the current subscription. I feel we should remove this line and just let users run `az account show`.

Reply:
removed